ci/install_geos.sh

```diff
@@ -32,14 +32,18 @@ prepare_geos_build_dir(){
 
 build_geos(){
     echo "Installing cmake"
-    pip install cmake ninja
+    pip install cmake
 
     echo "Building geos-$GEOS_VERSION"
     mkdir build
     cd build
-    cmake -GNinja -DCMAKE_BUILD_TYPE=Release -DBUILD_SHARED_LIBS=ON -DCMAKE_INSTALL_PREFIX=$GEOS_INSTALL ..
-    cmake --build .
-    # ctest .
+    # Use Ninja on windows, otherwise, use the platform's default
```

Comment:
Is there a reason for not doing this differently depending on the platform?

Reply:
Cmake happens te pick a working default on the posix platforms. On windows, it picks the “Visual Studio 2019” generator, which fails.

Reply:
you can set an environment variable [`CMAKE_GENERATOR=Ninja`](https://cmake.org/cmake/help/latest/envvar/CMAKE_GENERATOR.html) (either here or in the YAML file) for windows to override the default. Also, GEOS' default for `BUILD_SHARED_LIBS` for is `ON`, so there is no need for the following if/else/fi.

Reply:
Thanks for the feedback @mwtoews I actually tried that in this commit: https://github.com/pygeos/pygeos/pull/387/commits/c471ade8dde611dd901d7ccf77425c8062b6566b but it didn’t seem to work. I didn’t want to spend too much time on that so I went for the if … else … fi. Can you discover what’s wrong in that commit?

Reply:
The issue was setting the variable, which cannot have spaces, and needs to be exported. Give this a try:
```
    # Use Ninja on windows, otherwise, use the platform's default
    if [ "$RUNNER_OS" = "Windows" ]; then
        export CMAKE_GENERATOR=Ninja
    fi
```

Reply:
Thanks @mwtoews , I pushed the changes. Let's see what happens.

Reply:
It worked, only now a new issue popped up. So I removed the “skip building test” option which apparently broke the GEOS 3.8 build.